### ISSUE TITLE
The CanTieredAlloc test is too slow

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#547]
Originally reported of Google Code with ID 491.

```
The CanTieredAlloc unit test, part of the HTools/Cluster test suite is much slower than all the other tests.

It would be convenient to make it faster.
```
Originally added on 2013-06-19 11:33:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
As in the current state (with the pruning of commit f9e7e33) the run time of the test is bearable, it's no longer blocking. Therefore rescheduling to later release.
```
Originally added on 2013-06-28 07:13:03 +0000 UTC.

Added to Milestone: Release2.10
Removed from Milestone: Release2.8

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-01-28 09:04:17 +0000 UTC.

Added to Milestone: Release2.12
Removed from Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
Postponing non-blocking bugs
```
Originally added on 2014-06-03 12:04:01 +0000 UTC.

Added to Milestone: Release2.15
Removed from Milestone: Release2.12

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 76d6ca612002d0491c5942814a4356ad33835238
Author: Klaus Aehlig <aehlig@google.com>
Date:   Wed Mar 25 17:49:17 2015 +0100

    In CanTieredAlloc test set IPolicy
    
    Changing the test to allocate on nodes with a non-trivial
    instance policy has two advantages.
    
    - We test in a more realistic (and also more challenging)
      environment.
    
    - Once the lower limit of the ipolicy is reached no more
      allocations are possible, thus having fewer shrinking
      rounds---and hence speeding up the test by a factor of 4.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Hrvoje Ribicic <riba@google.com>

```
Originally added on 2015-03-26 14:22:47 +0000 UTC.

Added to Milestone: Release2.10
Removed from Milestone: Release2.15

**Comment by Ganeti-Issues-Migrator** [comment]
```
The time is now 25 seconds. Closing the bug; feel free to reopen if this
is still too slow.
```
Originally added on 2015-03-26 14:23:59 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:40:20 +0000 UTC.

Changed State: Released
